spring-boot-project/spring-boot-cli/src/main/java/org/springframework/boot/cli/command/init/ProjectGenerationRequest.java

```diff
@@ -191,7 +191,7 @@ void setPackageName(String packageName) {
 	}
 
 	/**
-	 * The type of project to generate. Should match one of the advertized type that the
+	 * The type of project to generate. Should match one of the advertised type that the
```

Comment:
TIL that US English doesn't use a z here.

Reply:
;)